flashinfer/autotuner/autotuner.py

```
@@ -1602,10 +1647,42 @@ def choose_one(
+                        _default_runner = runners[0]
+                        if "do_preparation" in runner_arg_names_map[_default_runner]:
+                            _default_runner(
+                                tensors, tactic=-1, do_preparation=True, **kwargs
+                            )
+                        try:
+                            default_time = self._profile_single_kernel(
+                                _default_runner, tensors, -1, tuning_config, **kwargs
+                            )
+                        except torch.cuda.OutOfMemoryError:
+                            raise
+                        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)